src/pages/Models/Details/ModelDetails.js

```diff
@@ -157,7 +157,7 @@ const ModelDetails = () => {
   return (
     <Layout sidebar={false}>
       <div className="model-details">
-        <InfoPanel />
+        <InfoPanel modelStatus={modelStatusData} />
```

Comment:
With the `redux` data flow change then this data will no longer be passed in as each component is responsible for fetching/requesting its own data.